### DISCUSSION TITLE
How do I test that a components property is equal to something?

### DISCUSSION THREAD

**Discussion by jadonbrownza** [livewire/livewire#3630] [1. Help]
Hi,

I've got a component that has a property defined as:
```php
public function getTotalCentsProperty() {
      xxx
      return $totalCents;
}
```

How can I test that the property is calculating correctly?

Apologies if it's a foolish question.

**Comment by jonneroelofs** [comment]
I think the example in the documentations may help you:
https://laravel-livewire.com/docs/2.x/testing

Or are you running into a specific problem?

**Comment by joshhanley** [comment]
@scheMeZa if I remember rightly, testing computed properties won't work with the standard tests.

Instead you can get an instance of the component and manually call the method and test it's result.

Here's how you'd get the instance and call the method

```php
Livewire::test(MyComponent::class)->instance()->getTotalCentsProperty()
```

Hope this helps!
